src/Servers/IIS/IntegrationTesting.IIS/src/IISDeploymentParameterExtensions.cs

```diff
@@ -37,7 +37,7 @@ public static void AddHttpsToServerConfig(this IISDeploymentParameters parameter
 
                     element.Descendants("access")
                         .Single()
-                        .SetAttributeValue("sslFlags", "Ssl, SslNegotiateCert");
+                        .SetAttributeValue("sslFlags", "None");
```

Comment:
I need to read into setting this to None. Setting this to none makes it seem like SSL is disabled for the entire site. I'm still seeing in the application that it is binding to https and using HTTP/2. Maybe it's using h2c?

Reply:
Nvrm, I think it's a poor doc. I'm seeing https on the server.

Reply:
So what does setting this to None actually do?

Reply:
It removes client cert validation I believe.